Source/Extensions/Blazorise.Components/SelectList.razor.cs

```diff
@@ -69,7 +103,23 @@ public Task Focus( bool scrollToElement = true )
         /// <summary>
         /// Currently selected item value.
         /// </summary>
-        [Parameter] public TValue SelectedValue { get; set; }
+        [Parameter]
+        public TValue SelectedValue
+        {
+            get => selectedValue;
+            set
+            {
+                if ( !selectedValue.Equals( value ) )
+                {
+                    selectedValue = value;
+                    SelectedValueChanged.InvokeAsync( selectedValue ).ConfigureAwait( false );
```

Comment:
You must not call InvokeAsync in synchronous code.

Reply:
I had a look at asp.net core components and the pattern they use is to trigger changed events use within a internal CurrentValueProperty. I propose to the same as Property can be changed from different sources. And there is no other way then raising InvokeAsync in synch code. At least I use no _ to indicate that code does not care of the task output.